packages/server/core/src/plugins/render/index.ts

```diff
@@ -133,8 +134,9 @@ function createRenderHandler(
     const nodeReq = c.env.node?.req;
 
     const res = await render(request, {
-      logger,
       nodeReq,
+      monitors,
+      logger,
```

Comment:
Why does the render function need to take logger and metrics as inputs?

Reply:
For void break change, we have export logger & metrics api to user by using `useRuntimeContext`